components/RapDetails.js

```diff
@@ -56,12 +68,52 @@ export default function RapDetails() {
     setCartTickets([...cartTickets, ticketData]);
   }
 
+  async function handleAddReview(event) {
+    event.preventDefault(event.target);
```

Comment:
`preventDefault` doesn't need the argument here.
```suggestion
    event.preventDefault();
```